content/pages/resources.md

```diff
@@ -0,0 +1,29 @@
+Title: Resources
+Category: pages
+Summary: Feminism101 and Other Resources
+
```

Comment:
I would maybe add a little blurb to this page specifying what it's for and what each of the sections are. If we're doing education here, not everyone will understand what "Feminism 101" means :)